### PR TITLE
Added inboxPrefix config option and changed the default value to _INBOX

### PR DESCRIPTION
**Pull request by dmccarthy-bluefin**
Added a new config inboxPrefix option and changed the default value to _INBOX. Related issue: https://github.com/basis-company/nats.php/issues/17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -267,3 +268,23 @@ Warning:       No code coverage driver available
 
 nekufa@fasiga ~ % cat /proc/cpuinfo | grep i5
 model name  : Intel(R) Core(TM) i5-4670K CPU @ 3.40GHz
+```
+
+## Configuration Options
+
```
Comment (adam-lmi) [inline README.md:274]: Could you please include all the existing config options, including jwt and nkey?

Comment (dmccarthy-bluefin) [reply README.md:274]: I'm not 100% sure how they work. Are they alternatives to CredentialsParser?

Comment (adam-lmi) [reply README.md:274]: CredentialsParser just returns a config array with nkey and/or jwt, but they can be defined directly

https://docs.nats.io/running-a-nats-service/configuration/securing_nats/auth_intro/jwt

JWT: JSON Web Token ( default null )
NKey: Ed25519 based public key signature ( default null )

Comment (dmccarthy-bluefin) [reply README.md:275]: I've added entries for jwts and nkeys to the latest.

**Pull request merged**